### PR TITLE
display build target info

### PR DESCRIPTION
**Pull request by Arthurm1**
see https://github.com/scalameta/metals-vscode/pull/660

### CODE REVIEW

File: src/interfaces/TreeViewProtocol.ts [hunk 1]
```diff
@@ -36,6 +36,8 @@ export interface MetalsTreeViewNode {
    *   this node to see the children.
    */
   collapseState?: "collapsed" | "expanded";
+  /** used to contribute item specific actions in the tree */
+  contextValue?: string;
```
Comment (tgodzik) [inline src/interfaces/TreeViewProtocol.ts:40]: Same question as in the vs code PR, do we need it only for VS Code or also for other clients?

**Review by tgodzik — CHANGES_REQUESTED** [review]
Could you rebase and we should be good to merge. I would just want to know if it were possible to avoid adding `contextValue`

File: src/interfaces/TreeViewProtocol.ts (hunk 1 above)
Comment (Arthurm1) [reply src/interfaces/TreeViewProtocol.ts]: I've removed ContextValue

**Comment by Arthurm1** [comment]
@tgodzik please merge if you're happy

**Review by tgodzik — APPROVED** [review]
LGTM!

**Pull request merged**